### PR TITLE
Add types declaration file

### LINKED ISSUE #1: TypeScript type declaration file
If you want I can write a TypeScript type declaration file for this package so that TS developers can utilise the type-checking feature of the language as they use this package. Let me know if you are interested.

### PR DESCRIPTION
**Pull request by geopic**
Resolves #1. Let me know if there are any issues.

### CODE REVIEW

File: lib/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+declare module 'dify-bin' {
+  import BManager from 'bin-manager';
+
+  export default function dify(images: string[], callback: Function): typeof BManager;
```
Comment (jihchi) [inline lib/index.d.ts]: It would be more clear if we change `images: string[]` to `arguments: string[]`.

**Review by jihchi — COMMENTED** [review]
Appreciate! thanks for the contribution!

File: lib/index.d.ts (hunk 1 above)
Comment (geopic) [reply lib/index.d.ts]: Noted, I will change that now :+1:

**Comment by jihchi** [comment]
Merged! thanks again!

**Comment by geopic** [comment]
No problem :smile:

**Pull request merged**